src/Pie.js

```diff
@@ -72,53 +133,63 @@ const RoundDividers = ({ paintedSections, dividerSize, width, radius, background
   );
 };
 
-const getArcAngle = (percentage) => percentage / 100 * 360;
-const shouldShowDivider = (sections, dividerSize) => sections.length > 1 && !Number.isNaN(dividerSize);
+// These circles clean up the strokes left over from the bezian curves
+const CleanUpCircles = ({dimensions, backgroundColor, visible}) => {
+  const { radius, innerRadius, width} = dimensions;
+  const innerBackgroundPath = createPath(radius, radius, innerRadius - ((width) / 2), 0, 360);
+  const outerBackgroundPath = createPath(radius, radius, radius + ((width)) / 2, 0, 360);
+  if((width) < 100 && visible){
+    return (<>
+      <Shape
+          d={innerBackgroundPath}
+          stroke={backgroundColor}
+          strokeWidth={width}
+      />
+      <Shape
+        d={outerBackgroundPath}
+        stroke={backgroundColor}
+        strokeWidth={width}
+      />
+    </>)
+  }
+  return null;
+}
 
 const Pie = ({ sections, radius, innerRadius, backgroundColor, strokeCap, dividerSize }) => {
-  const width = radius - innerRadius;
-  const backgroundPath = createPath(radius, radius, radius - width / 2, 0, 360);
-  const shouldShowRoundDividers = !!dividerSize && strokeCap === 'round';
-  let startValue = 0;
+  strokeCapForLargeBands = dividerSize > 0 || strokeCap == 'butt' ? 'butt' : 'butt';
+  const shouldShowRoundDividers = strokeCap === 'round';
   let paintedSections = [];
-  const showDividers = shouldShowDivider(sections, dividerSize);
+  
+  // This is the width for the arc
+  const width = radius - innerRadius;
+  const dimensions = {
+    radius: radius,
```

Comment:
why not use shorthand here?

Reply:
shorthand as in dim instead of dimensions?

Reply:
Sorry I mean `dimensions = { radius, innerRadius, width, ... }`

Reply:
ohh I got you didn't realize we could do that